application/actions/auth.go

```diff
@@ -218,18 +227,35 @@ func meetingAuthRequest(c buffalo.Context, authEmail string, extras map[string]i
 			extras)
 	}
 
-	// find org for auth config and processing
-	org, userOrgs, err := getOrgAndUserOrgs(authEmail, c)
+	// If the user already has organizational auth options, use them
```

Comment:
There is also a difference in the call to `finishAuthRequestForSocialUser()` in the middle of `authRequest()`. I'm not sure if refactoring it would actually end up being preferable.

Reply:
In my opinion, it would be preferable because of the 20-30 lines of duplicate code, but you're the one who spent all the time figuring this out. I'll leave it in your court.